packages/graphql/src/translate/create-projection-and-params.ts

```diff
@@ -185,7 +185,14 @@ export default function createProjectionAndParams({
                     const direction = getCypherRelationshipDirection(relationField, field.args);
 
                     const nestedProjection = new Cypher.RawCypher((env) => {
-                        const nestedProj = recurse.projection.getCypher(env).replace(/{|}/gm, "").trim();
+                        // The nested projection will be surrounded by brackets, so we want to remove
+                        // any linebreaks, and then the first opening and the last closing bracket of the line,
+                        // as well as any surrounding whitespace.
+                        const nestedProj = recurse.projection
+                            .getCypher(env)
+                            .replaceAll("\n", " ")
+                            .replace(/^\s*{\s*|\s*}\s*$/g, "");
```

Comment:
`\s` already takes white space into account, so the first step of replacing `\n` into `" "` should not be needed (unless there is another reason for this addition that I have not though of :thinking:?

Additionally, I would consider either changing this to `replaceAll` (as it forces the regex to be global and better denotes what we try to do). Whatever you feel is nicer

 ```suggestion
                             .replaceAll(/^\s*{\s*|\s*}\s*$/g, "");
```

Another suggestion on this regex, could be adding parenthesis to denote the separate alternatives:

```regex
(^\s*{\s*)|(\s*}\s*$)
```

This is mainly for readability, and making this regex less prone to errors if modified in the future, but I'm happy with your current regex as well if you feel this is not a good idea

Reply:
Hi, if I'm not mistaken, that would only happen if the multiline option (`/m`) is set in the regex, which is not the case so it should be safe to let the regex take care of that

Reply:
Just checked and you're completely right, my bad. Removing that line and pushing again.

Reply:
no worries, thanks :smile: